LibreNMS/OS/Fs.php

```diff
@@ -55,6 +55,8 @@ public function discoverProcessors()
         }
         // Tests OID from SWITCH MIB.
         $processors_data = snmpwalk_cache_oid($this->getDevice(), 'ssCpuIdle', [], 'SWITCH', 'fs');
+
+        $processors = [];
```

Comment:
Not good. If you do so, you potentially forget the processors that were discovered in the previous foreach. 
New commit on its way.